Sources/AppStoreConnectCLI/Model/ProcessingState.swift

```diff
@@ -0,0 +1,15 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import ArgumentParser
+import Foundation
+
+enum ProcessingState: String, CaseIterable, CustomStringConvertible {
+    case processing = "PROCESSING"
+    case failed = "FAILED"
+    case invalid = "INVALID"
+    case valid = "VALID"
+
+    public var description: String {
+        rawValue.lowercased()
+    }
+}
```

Comment:
Is this file being used?

Reply:
I didn't add this..it was added by whoever started this PR